### ISSUE TITLE
Add unfilled star icon

### ISSUE THREAD

**Issue by phoebe-leung** [StackExchange/Stacks-Icons#402]
We need an unfilled version of the star icon so that we can pass accessibility. We currently only have the filled icon, and so unfilled vs filled is only a color change which fails accessibility. 

I've added an unfilled icon into the Figma Icon & Spot Illustration file here: https://www.figma.com/design/NxAqQAi9i5XsrZSm1WYj6tsM/Icons---Spot-Illustrations?m=auto&node-id=13940-15&t=w7S0DQoi4SgTAH3j-1

**Comment by dancormier** [comment]
I've created a PR that adds these icons to the library here https://github.com/StackExchange/Stacks-Icons/pull/403.

Prior to creating this PR, I also updated the [README](https://github.com/StackExchange/Stacks-Icons/blob/production/README.md#adding-a-published-icon-to-this-library) to include instructions for publishing an icon in Figma in order to expose it for consumption via the Figma API.

**Comment by phoebe-leung** [comment]
Thanks Dan. I will turn the icon into a component next time - thank you!
